src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search search__big"
+      data-qa="big"
+      >
```

Comment:
Fix code style everywhere
```suggestion
    <form
      action="#"
      class="search search__big"
      data-qa="big"
    >
```

Reply:
I am fixed